app/src/main/java/org/oppia/app/player/content/ContentCardAdapter.kt

```diff
@@ -0,0 +1,106 @@
+package org.oppia.app.player.content
+
+import android.content.Context
+import android.text.Html
+import android.text.Spannable
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.databinding.library.baseAdapters.BR
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.content_card_items.view.*
+import org.oppia.app.R
+import org.oppia.app.databinding.ContentCardItemsBinding
+import org.oppia.app.databinding.RightInteractionCardItemBinding
+import org.oppia.data.backends.gae.model.GaeSubtitledHtml
+import org.oppia.util.data.URLImageParser
+
+/** Adapter to bind the htm-contents to the recyclerview. */
+class ContentCardAdapter(
+  internal var context: Context,
+  contentList: MutableList<GaeSubtitledHtml>
+) :
+  RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+
+  private val VIEW_TYPE_CONTENT = 1
+  private val VIEW_TYPE_RIGHT_INTERACTION = 2
```

Comment:
it is learner's interaction to be displayed on right side of the view

Reply:
We shouldn't use positional language like 'left/right' since they may be reversed in RTL layouts.

Beyond that, I'm not sure I understand the directionality here. Per the [mocks](https://xd.adobe.com/spec/bee1fced-33bd-4f69-6a89-2c30fe1c4f65-c8d0/screen/4744bfae-84be-44fa-a27d-9a252fdfc2f9/Lesson-1-a-i-Exploration-Player-Fraction-Input-No-) the interaction appears below the content area.